devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -300,6 +303,23 @@ def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulat
 	}
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.configurationv2.ConfigurationReport cmd) {
+	def checkVal
+	// the last message sent by configure is a configuration get, so if we get a report, we succeeded in transmission
+	// and if the parameter 1 values match what we expect, then the configuration probably succeeded
+	if (isZipato()) {
+		checkVal = 3
+	} else if (isYale()) {
+		checkVal = state.alarmLength
```

Comment:
I don't believe that settings are accessible through state like that. I think you can either use `alarmLength` or `settings.alarmLength`.

Reply:
it's used further up in `configure`

Reply:
Ah! I should have known that.... Seeing as I have touched this code... 🤦‍♂️